src/manifest.c

```diff
@@ -233,6 +233,9 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		if (strncmp(line, "contentsize:", 12) == 0) {
 			contentsize = strtoull(c, NULL, 10);
 		}
+		if (strncmp(line, "actions:", 8) == 0) {
+			post_update_action = strdup(c);
```

Comment:
We could explore that path when/if enabling swupd to follow the actions itself instead of just printing a warning. I am inclined to say the actions: are automatically validated if the signature verification passes, and we should also not ever publish corrupt manifests to begin with, so perhaps the validation would happen before it ever gets to clients?